core/includes/common.inc

```diff
@@ -4339,19 +4344,23 @@ function backdrop_delete_file_if_stale($uri) {
  * @return
  *   The cleaned identifier.
  */
-function backdrop_clean_css_identifier($identifier, $filter = array(' ' => '-', '_' => '-', '/' => '-', '[' => '-', ']' => '')) {
+function backdrop_clean_css_identifier($identifier, $filter = array(' ' => '-', '_' => '-', '[' => '-', ']' => '')) {
```

Comment:
```suggestion
function backdrop_clean_css_identifier($identifier, $filter = array(' ' => '-', '_' => '-', '/' => '-', '[' => '-', ']' => '', '@' => '', ':' => '', '.' => '')) {
```

To maintain backwards-compatibility we keep the slash, and add period, colon and "at".

Reply:
If the at-sign characters are removed, then there would not be any difference between identifiers like `@home` and `home`, when they should different identifiers.

Reply:
@avpaderno but we're now aiming to keep this backwards compatible and let the alter hook allow for that difference. Currently the `@` is always removed so that's what I'm attempting to do.

Reply:
How could a hook implementation know the identifier was `@home` instead of `home`, if `backdrop_clean_css_identifier()` removes the at-sign character?